src/libraries/System.Text.RegularExpressions/src/System.Text.RegularExpressions.csproj

```diff
@@ -51,6 +50,7 @@
     <Compile Include="$(CommonPath)System\HexConverter.cs" Link="Common\System\HexConverter.cs" />
     <Compile Include="$(CoreLibSharedDir)System\Collections\Generic\ValueListBuilder.cs" Link="Common\System\Collections\Generic\ValueListBuilder.cs" />
     <Compile Include="$(CommonPath)System\Text\ValueStringBuilder.cs" Link="Common\System\Text\ValueStringBuilder.cs" />
+    <Compile Include="$(CommonPath)System\Text\ValueStringBuilder.Pop.cs" Link="Common\System\Text\ValueStringBuilder.Pop.cs" />
```

Comment:
I think this was supposed to be ValueListBuilder.pop. I'll fix that on the changes I'm about to push.